python/felis/metadata.py

```diff
@@ -286,7 +286,7 @@ def build_column(self, column_obj: datamodel.Column) -> Column:
             comment=description,
             autoincrement=autoincrement,
             nullable=nullable,
-            server_default=default,
+            server_default=text(str(default)) if default else None,
```

Comment:
> Can default be an empty string? And if column type is a string then you probably do not want `text()` it because that will render it without quotes in DDL.

It probably can be an empty string. I can add some logic to check for this and set accordingly.

Reply:
> Can default be an empty string? And if column type is a string then you probably do not want `text()` it because that will render it without quotes in DDL.

I determined that empty or whitespace-only strings will actually be interpreted by Pydantic as `None` values due to the `str_strip_whitespace` setting currently in use, so on default will be set in the DDL. This seems reasonable since there is not a good use case for these values being the default for a string column.